bindings/python/crocoddyl/core/actuation-base.hpp

```diff
@@ -44,6 +44,10 @@ class ActuationModelAbstract_wrap : public ActuationModelAbstract, public bp::wr
   }
 
   boost::shared_ptr<ActuationDataAbstract> default_createData() { return this->ActuationModelAbstract::createData(); }
+  void get_actuated(const boost::shared_ptr<ActuationDataAbstract>& data, Eigen::Ref<Eigen::VectorXd>& u) {
```

Comment:
Is this a pure virtual method? If not, this is not the proper way of bind it.

Reply:
Thanks for the help in fixing it.